geopyspark/geotrellis/rdd.py

```diff
@@ -433,6 +433,14 @@ def rasterize(cls, geopysc, rdd_type, geometry, extent, crs, cols, rows,
 
         return cls(geopysc, rdd_type, srdd)
 
+    @classmethod
+    def euclidean_distance(cls, geopysc, geometry, source_crs, zoom):
+        if isinstance(source_crs, int):
+            source_crs = str(source_crs)
+
+        srdd = geopysc._jvm.geopyspark.geotrellis.SpatialTiledRasterRDD.euclideanDistance(geopysc.sc, dumps(geometry), source_crs, zoom)
```

Comment:
I know you said that any geometry can be used, but that certain ones would be very slow. Do you think it'd be worth have a warning message come up if the user is trying to use a suboptimal geometry?